### PR TITLE
Ensure that long usernames don't push out replies

### PR DESCRIPTION
**Pull request by gtrufitt**
## What does this change?

Usernames will be truncated if they are too long and don't have enough space, instead of before where they pushed the comments container too wide.

They truncate in the most space effective way possible, always showing as much of each username as is practical.

### Before
image

### After
Screen Shot 2020-07-31 at 11 35 38
Screen Shot 2020-07-31 at 11 35 30
Screen Shot 2020-07-31 at 11 34 46

### CODE REVIEW

File: src/components/Comment/Comment.tsx [hunk 1]
```diff
@@ -211,6 +212,43 @@ const negativeMargin = css`
...
+
+const cssReplyDisplayName = css`
+    ${until.mobileLandscape} {
+        ${cssTextOverflowElip}
+        width: 100%;
+        max-width: fit-content;
+    }
+`;
+
+const cssReplyToDisplayName = css`
```
Comment (jfsoul) [inline src/components/Comment/Comment.tsx:244]: I guess these are accurate names, but a bit too similar at first glance!

File: src/index.tsx [hunk 2]
```diff
@@ -38,6 +38,7 @@ const IndexPageWrapper = () => {
                 </a>
                 .
             </p>
+            <style>{`body { padding: 0 10px;}`}</style>
```
Comment (jfsoul) [inline src/index.tsx:41]: This necessary to avoid the overflow or just an additional unrelated change?

Comment (gtrufitt) [reply src/index.tsx:41]: Yeah, unrelated, the index.tsx is used *just* for local testing (it doesn't get bundled for the app)
